app/views/observer/_form_observations.html.erb

```diff
@@ -141,12 +141,17 @@
 </div>
 <% inject_javascript_at_end %(
   jQuery(document).ready(function() {
+//hide initially if has specimine box isn't checked
```

Comment:
Fixing a syntax bug in the javascript that caused javascript error which prevent the toggling of herbarium specimen fields from working properly.